frontend/src/component/commandBar/RecentlyVisited/CommandResultGroup.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+    Icon,
+    List,
+    ListItemButton,
+    ListItemIcon,
+    ListItemText,
+    styled,
+    Typography,
+} from '@mui/material';
+import { Link } from 'react-router-dom';
+import type { Theme } from '@mui/material/styles/createTheme';
+
+const listItemButtonStyle = (theme: Theme) => ({
+    borderRadius: theme.spacing(0.5),
+    borderLeft: `${theme.spacing(0.5)} solid transparent`,
+    '&.Mui-selected': {
+        borderLeft: `${theme.spacing(0.5)} solid ${theme.palette.primary.main}`,
+    },
+});
+
+const StyledTypography = styled(Typography)(({ theme }) => ({
+    fontSize: theme.fontSizes.bodySize,
+    padding: theme.spacing(0, 3),
+}));
+
+const StyledListItemIcon = styled(ListItemIcon)(({ theme }) => ({
+    minWidth: theme.spacing(4),
+    margin: theme.spacing(0.25, 0),
+}));
+
+const StyledListItemText = styled(ListItemText)(({ theme }) => ({
+    margin: 0,
+}));
+
+export interface CommandResultGroupItem {
+    name: string;
+    link: string;
+}
+
+interface CommandResultGroupProps {
+    icon: string;
+    groupName: string;
+    items: CommandResultGroupItem[];
+}
+
+export const CommandResultGroup = ({
```

Comment:
This can be reused also for projects and we need to see if we can use it also for pages.

Reply:
Yeah I think we can use this for recently visited as well